serveradmin/servershell/static/js/servershell/result.js

```diff
@@ -84,24 +84,40 @@ get_row_html = function(object, number) {
                     let to_delete = change.remove.join(', ');
                     let value = object[attribute_id].filter(v => !to_delete.includes(v)).join(', ');
 
-                    cell.html(`${value} <del>${to_delete}</del> <u>${to_add}</u>`)
+                    cell.text(value);
+
+                    let del = $('<del></del>');
```

Comment:
Btw, for creating just an empty html elements you don't need the closing tag:
``` js
$('<del>')
```
is sufficient already.

Reply:
Ah neat. I will adjust the code.